src/utilities/sequence-editor/to-seq-json.ts

```diff
@@ -517,6 +559,33 @@ function parseVariables(
   }) as VariableDeclarationArray;
 }
 
+function parseAllowableRanges(text: string, rangeNode: any): { max: number; min: number }[] {
+  return text
+    .slice(rangeNode.from, rangeNode.to)
+    .split(',')
+    .map(range => {
+      const rangeMatch = /^([-+]?\d+)?(\.\.\.)([-+]?\d+)?$/.exec(range.replaceAll('"', '').trim());
+      if (rangeMatch) {
+        const [, min, , max] = rangeMatch;
+        const maxNum = !isNaN(Number(max)) ? Number(max) : Infinity;
```

Comment:
Suuuuper small (and probably inconsequential) performance comment: 

`isNaN` goes through the extra step of attempting to convert the argument to a number before determining if it's a number or not. There is a `Number.isNaN` (that is not the same as just `isNaN`) that basically does an equality check to see if the argument is literally `NaN` (which Number potentially outputs) so it is ever so slightly more performant in this case.

To save more on performance, you could also save the output of `Number(max)` prior to the check so that it's only converted once.
i.e.
```
const parsedMaxNum = Number(max);
...
```

Reply:
To be even MOAR performant, the `map` + `filter` could be changed to a `reduce` so that you have control on the final array, but I'll let you decide if you want to go that far.

Reply:
I addressed the first part and kinda want to get this into the release today so can I ignore the MOAR performant request?

Reply:
Yeah that's fine. This doesn't need to be super performant, right?

Reply:
It doesn't need to be 'ludicrious-mode' performant, it is pretty performant as of now